### ISSUE TITLE
Could not find module in path: '@iconify/vue2' when using @vue/cli@5 and @iconify/vue2@1.1.0

### ISSUE THREAD

**Issue by ModyQyW** [iconify/iconify#93]
[codesandbox](https://codesandbox.io/s/keen-hertz-09nh1?file=/src/App.vue)

```sh
ModuleNotFoundError
Could not find module in path: '@iconify/vue2' relative to '/src/App.vue'
```

It is fine when using @vue/cli@5 and @iconify/vue2@1.0.0. It only happens when using @vue/cli@5 and @iconify/vue2@1.1.0.

**Comment by cyberalien** [comment]
All files are there, `exports` in package.json are set correctly, there are `main` and `module` fields for bundlers that do not support `export` field, so it is a bug in bundler sandbox is using.

**Comment by cyberalien** [comment]
Opened bug report at sandbox repo: https://github.com/codesandbox/codesandbox-client/issues/6296

**Comment by ModyQyW** [comment]
I can get the same error when I dev locally, using @vue/cli@5.0.0-rc.1 and @iconify/vue2@1.1.0. I think it is not a sandbox problem, but a @vue/cli problem, maybe?

**Comment by ModyQyW** [comment]
I think I know where the problem is.

In package.json:

```json
{
	...,
	"exports": {
		".": {
			"import": "./dist/index.mjs",
			"types": "./dist/index.d.ts",
			"default": "./dist/index.js"
		},
		"./offline": {
			"import": "./dist/offline.mjs",
			"types": "./dist/offline.d.ts",
			"default": "./dist/offline.js"
		}
	},
	...
}
```

But no `index.mjs` / `index.d.ts` / `index.js` in `dist`, only `iconify.mjs` / `iconify.d.ts` / `iconify.js`.

**Comment by ModyQyW** [comment]
I have opened a pr https://github.com/iconify/iconify/pull/94.

**Comment by cyberalien** [comment]
Doh. Thanks a lot for noticing!

**Comment by cyberalien** [comment]
Fixed, published version 1.1.1 and tested it with your sandbox. Everything is working again.

**Comment by ModyQyW** [comment]
Thanks! :D
